### PR TITLE
Add tests for receptor_type with CollocatedSynapses

### LINKED ISSUE #1805: Add tests for receptor_types and CollocatedSynapses
Once #1645 is merged, we should add some tests for `receptor_types` and `CollocatedSynapses`.

### PR DESCRIPTION
**Pull request by stinebuu**
Add tests of `receptor_type` for `CollocatedSynapses`.

Fixes #1805.

### CODE REVIEW

**Review by hakonsbm — APPROVED** [review]
LGTM!

File: pynest/nest/tests/test_multiple_synapses.py [hunk 1]
```diff
@@ -292,6 +292,51 @@ def test_MultipleSynapses_make_symmetric(self):
...
+        node = nest.Create('iaf_psc_alpha')
+
+        syn_spec = nest.CollocatedSynapses({'synapse_model': 'stdp_synapse',
+                                            'weight': 5.,
+                                            'receptor_type': 2},
+                                           {'weight': 1.5, 'receptor_type': 7},
+                                           {'synapse_model': 'stdp_synapse', 'weight': 3, 'receptor_type': 5})
+
+        nest.Connect(src, trgt, 'one_to_one', syn_spec=syn_spec)
+        nest.Connect(node, node)  # should have receptor 0
```
Comment (clinssen) [inline pynest/nest/tests/test_multiple_synapses.py:311]: What is described in this comment is not checked in an assert.

**Review by clinssen — CHANGES_REQUESTED** [review]

File: pynest/nest/tests/test_multiple_synapses.py (hunk 1 above)
Comment (stinebuu) [reply pynest/nest/tests/test_multiple_synapses.py:311]: Done!

**Review by clinssen — APPROVED** [review]
Cheers!

**Pull request merged**